lib/foreman_discovery/engine.rb

```diff
@@ -107,6 +103,7 @@ class Engine < ::Rails::Engine
           :view_organizations,
           :view_locations,
           :view_hosts,
+          :view_operatingsystems,
```

Comment:
Nice, I need to test if this adds the permission to an existing role, otherwise we'd need a migration like:

```
class AddOrglocPermissions < ActiveRecord::Migration
  def up
    [Role.find_by_name("Discovery Manager"), Role.find_by_name("Discovery Reader")].each do |r|
      next if r.nil?
      r.add_permissions!(["view_organizations", "view_locations"])
    end
  end

  def down
  end
end
```